activesupport/lib/active_support/file_evented_update_checker.rb

```diff
@@ -0,0 +1,60 @@
+require 'listen'
+
+module ActiveSupport
+  class FileEventedUpdateChecker
+    attr_reader :listener
+    def initialize(files, directories={}, &block)
```

Comment:
Please add a blank line before the method definition.

Also please add a space before and after the `=`.